src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -369,4 +373,57 @@ describe('StoryPromo Container', () => {
       });
     });
   });
+
+  describe('given there is a CPS MAP block with a media error', () => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
+    [
+      {
+        item: mapWithMediaError,
+        platform: 'amp',
+      },
+      {
+        item: mapWithMediaError,
+        platform: 'canonical',
+      },
```

Comment:
I have left this structure so that we can add future block types as we support them. e.g. Optimo blocks without media.